src/main/java/step3/RandomPolicy.java

```diff
@@ -0,0 +1,26 @@
+package step3;
+
+import java.util.Random;
+
+public class RandomPolicy implements Policy {
+
+    private final Random random = new Random();
+    private boolean attemptsSuccess;
+
+    public boolean isAttemptsSuccess() {
+        return attemptsSuccess;
+    }
+
+    @Override
+    public int apply(int position) {
+        if (isSuccess()) {
+            return position += 1;
+        }
+        return position;
+    }
+
+    private boolean isSuccess() {
+        attemptsSuccess = random.nextInt(10) >= 4;
+        return attemptsSuccess;
```

Comment:
```suggestion
        return random.nextInt(10) >= 4;
```
1. 위와 같이 쓰는게 좀더 깔끔하지 않을까요?
2. 4라는 값이 어떤걸 의미하는지 저는 처음 보는 입장에서 알기가 어려운거 같아요. ㅎㅎ